spec/forms/hyrax/forms/permission_template_form_spec.rb

```diff
@@ -195,6 +220,95 @@
     end
   end
 
+  describe "#validate_visibility_combinations" do
+    let(:admin_set) { create(:admin_set) }
+    let(:permission_template) { create(:permission_template, admin_set_id: admin_set.id) }
+    let(:form) { described_class.new(permission_template) }
+
+    context "validate all release option attribute combinations" do
+      def self.expect_attributes_to_be_valid(attributes = {})
+        it "will be valid? for #{attributes.inspect}" do
+          attributes = ActionController::Parameters.new(**attributes).permit!
+          expect(form.update(attributes)).to eq(content_tab: "visibility", updated: true)
+        end
+      end
+
+      def self.expect_attributes_not_to_be_valid(attributes = {})
```

Comment:
This is an undesirable spot to define a method.  It looks like it is scoped to this context block, but in execution it is not.  Can we rework these using rspec `shared_examples`?  I can propose a revision or assist if that is unfamiliar.